### ISSUE TITLE
3.x Support?

### ISSUE THREAD

**Issue by acbart** [platipy/spyral#45]
Investigate whether or not Spyral works in 3.x, and what changes would be necessary to transition to it.

**Comment by icarito** [comment]
This would be dependent of Sugar porting to 3.x which is work in early process.

**Comment by acbart** [comment]
Next order of business will be to catch up on Sugar then - I haven't tracked their ecosystem, are they still developing?

**Comment by icarito** [comment]
Yes, Sugar is mantained and alive. I believe the first step would be to port the [sugargame2](https://github.com/platipy/sugargame2) bits to GTK3.
